test/features/comments.ts

```diff
@@ -274,4 +274,25 @@ export default function supportsComments(format: FormatFn, opts: CommentsConfig
       `);
     });
   }
+
+  it('should handle comma in comments with comaPosition:before', () => {
```

Comment:
This test should be in [test/options/commaPosition.ts](https://github.com/sql-formatter-org/sql-formatter/blob/master/test/options/commaPosition.ts) file.